src/test/java/se/sundsvall/billingdatacollector/integration/opene/mapper/kundfakturaformular/InternFakturaTests.java

```diff
@@ -0,0 +1,40 @@
+package se.sundsvall.billingdatacollector.integration.opene.mapper.kundfakturaformular;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import se.sundsvall.billingdatacollector.support.annotation.UnitTest;
+
+@UnitTest
```

Comment:
What is this? Purpose?

Reply:
I think Jonas created it for always setting the same/correct profile for unit tests.

Reply:
Then i would suggest to remove this annotation on places where you don't need a SpringBoot context.